webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_view_settings.py

```diff
@@ -0,0 +1,131 @@
+from typing import List
+
+import pandas as pd
+import webviz_core_components as wcc
+from dash import Input, Output, callback, html
+from dash.development.base_component import Component
+from webviz_config.utils import StrEnum, callback_typecheck
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC
+
+
+class ViewSettings(SettingsGroupABC):
+    class IDs(StrEnum):
+        REFERENCE = "reference"
+        SCALE = "scale"
+        SENSITIVITIES = "sensitivities"
+        RESET_BUTTON = "reset-button"
+        PLOT_OPTIONS = "plot-options"
+        LABEL = "label"
+
+    def __init__(
+        self,
+        realizations: pd.DataFrame,
+        reference: str = "rms_seed",
+        allow_click: bool = False,
+    ) -> None:
+        super().__init__("View Settings")
+        unique_realizations = realizations["SENSNAME"].unique()
+        self.sensnames = list(unique_realizations)
+        self.scales = [
+            "Relative value (%)",
+            "Relative value",
+            "True value",
+        ]
+        self.plot_options = [
+            {
+                "label": "Remove sensitivites with no impact",
+                "value": "Remove sensitivites with no impact",
+            },
+            {
+                "label": "Show realization points",
+                "value": "Show realization points",
+            },
+            {
+                "label": "Color bars by sensitivity",
+                "value": "Color bars by sensitivity",
```

Comment:
These are not shared settings - as they only affect the plot object in plot view. Move to `views/plot_view/_settings.py` file. NB: Defined a `StrEnum` for options values - to prevent hard coded strings.